src/main/kotlin/com/epam/carapi/entity/Dealer.kt

```diff
@@ -0,0 +1,14 @@
+package com.epam.carapi.entity
+
+import javax.persistence.*
```

Comment:
It would be great to avoid the usage such imports as `package.name.*`.
The good way is to import only necessary classes/interfaces.

Reply:
In addition to the comment above I'd add that there's a setting in the InteliJ Idea to configure wildcard imports. Here's for Java, but the same works for Kotlin: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports